actions/go.mod

```diff
@@ -81,6 +81,29 @@ require (
 
 )
 
+require (
```

Comment:
due to these bumps, all of airgap should live in interoperability folder (not actions)